### ISSUE TITLE
missing some clear expectation for customer

### ISSUE THREAD

**Issue by anboudin** [MicrosoftDocs/dynamics-365-unified-operations-public#3575]
Hi, we received cases on settlement from customer that refers to this document.
Customer mentioned that solution is provided by Microsoft so it should work.

From what I know Microsoft support is based on 2 decimals. 
can we had somewhere? extension is customization and Microsoft provides global guidance. but all scenario should be tested and code adapted by customer. 
Scenario related to this extension will be considered as customization.

This is not clear on this page. Or I am wrong.

Kind regards
Annaick

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4be50ccc-ed32-eabe-c717-cf2b319d6d22
* Version Independent ID: 4f8d76c5-2ca7-b191-9317-15bfdbb7b3fc
* Content: [Extending decimal point precision for selected data types - Finance & Operations | Dynamics 365](https://docs.microsoft.com/en-us/dynamics365/fin-ops-core/dev-itpro/extensibility/decimal-point-precision?fbclid=IwAR1md9eJf0krcE-Y-cTPBkvhUWJ056TblPuWHgJRokbbE8WOmA1IjvzAUyo)
* Content Source: [articles/fin-ops-core/dev-itpro/extensibility/decimal-point-precision.md](https://github.com/MicrosoftDocs/Dynamics-365-Unified-Operations-Public/blob/main/articles/fin-ops-core/dev-itpro/extensibility/decimal-point-precision.md)
* Service: **dynamics-365-cross-app**
* Product: ****
* Technology: ****
* GitHub Login: @MichaelFruergaardPontoppidan
* Microsoft Alias: **mfp**

**Comment by josaw1** [comment]
Thank you for your feedback. @MichaelFruergaardPontoppidan can you please review?

**Comment by MichaelFruergaardPontoppidan** [comment]
Since the topic is placed under Extensibility (https://docs.microsoft.com/en-us/dynamics365/fin-ops-core/dev-itpro/extensibility/extensibility-home-page) and there is a lot of context about roles and expectations on those page, I would be hesitant to repeat in this topic.  And this topic is named "Extending ..."

We are outlining the default support for each of the EDTs explicitly: "Weight data can be maintained with a maximum of two decimals by default."

**Comment by anboudin** [comment]
It was just to clarify on same page "how to create extension for 3 decimals" that it is not FULL solution but some GUIDANCE that needs to be fully tested and investigated to avoid data corruption detected such as rounding at posting/settlement.
We have several support cases and customer mentioned that they follow MS suggestion. As provided by MS customers think it should work without bug.
If it is on same page, they cannot complain as per their responsibility. Microsoft only provide Guidance.

I get that it is part of extension information and I checked, it is clear extension is customization.
But should be added somewhere that suggestion is advised to help customer but customer is responsible for the code change.

We manage to explain customer but will be easier if on same page.

Cordialement/Best Regards
Annaick
Tel: +33 15775 7012

[MSFT_logo_Gray DE sized SIG1.png]

Team Manager | Marwa Ibrahim Ali  ***@***.******@***.***>)

**Comment by anboudin** [comment]
It was just to clarify on same page “how to create extension for 3 decimals” that it is not FULL solution but some GUIDANCE that needs to be fully tested and investigated to avoid data corruption detected such as rounding at posting/settlement. 
We have **several support cases** and customer mentioned that they follow MS suggestion. As provided by MS customers think it should work without bug.
If it is on same page, they cannot complain as per their responsibility. Microsoft only provide Guidance.

I get that it is part of extension information and I checked, it is clear extension is customization.
But should be added somewhere that suggestion is advised to help customer but customer is responsible for the code change.

I can share with you mail from customer or recording of meeting. they all mentioned the same.
example case 2109090060002971 (mail from customer)

_All this while the customer was doing transactions in both systems their legacy ERP and D365.  They were getting few intermittent issues and the user was hesitating to move on with D365.  Few were resolved by Microsoft thru updates. The customer decided that either they continue if D365 is stable or else they stop D365 and move on with their old system. That cut-off date is October 1st =  2021. They're using this system in 6 legal entities and 4 countries. A lot of intercompany, country-based transactions with 2, 3 decimals. 

MS claims the solution works with 3 decimals. Further to that based on the Microsoft direction/documentation, we changed it. It was not our decision to change. So why would partner do debugging so what is Microsoft Role? 

Need an escalation, we can come on a call to explain this. It is not a non-standard code that we have written it is provided by MS. We can remove that code provided by MS guidelines, then MS should be supporting the 3 decimals._
